code/modules/mob/living/carbon/human/body_accessories.dm

```diff
@@ -127,7 +127,6 @@ GLOBAL_LIST_INIT(body_accessory_by_species, list())
 	allowed_species = list("Wryn")
 
 //Moth
```

Comment:
```suggestion
//Moth wings
```

Reply:
that was changed to just be moth for a reason, the antennae were in that file also and had their own section, but was since removed. The remainder of species-specific body accessories are denoted by species name only.
I have no issues changing this, but there is precedent.